### PR TITLE
Integrations tests

### PR DESCRIPTION
**Pull request by AStox**
This PR adds liquidity pools integrations to the deployments and fork tests. For now, these include CFG and verUSDC, and the tests just verify the wards are set correctly on each.

### CODE REVIEW

File: test/fork/Fork.t.sol [hunk 1]
```diff
@@ -459,4 +469,24 @@ contract ForkTest is Test {
...
+    function testIntegrations() public {
+        if (vm.envOr("FORK_TESTS", false)) {
+            for (uint256 i = 0; i < deployments.length; i++) {
+                address cfg = _get(i, ".integrations.cfg");
+                address verUSDC = _get(i, ".integrations.verUSDC");
+                address root = _get(i, ".contracts.root");
+                address admin = _get(i, ".config.admin");
+                _loadFork(i);
+
+                if (verUSDC != address(32)) {
```
Comment (hieronx) [inline test/fork/Fork.t.sol:482]: Im' curious where the `address(32)` comes from. Probably worth creating an `UNSET_ADDRESS` constant for this

File: test/fork/Fork.t.sol [hunk 2]
```diff
@@ -459,4 +469,24 @@ contract ForkTest is Test {
...
+                    assertTrue(IAuth(verUSDC).wards(root) == 1);
+                    assertTrue(IAuth(IWrappedUSDC(verUSDC).memberlist()).wards(admin) == 1);
+                }
+                if (cfg != address(32)) {
+                    assertTrue(IAuth(cfg).wards(root) == 1);
```
Comment (hieronx) [inline test/fork/Fork.t.sol:487]: Actually some other easy sanity checks we can add:
- `name = Centrifuge`
- `symbol = CFG` except on Ethereum where it's `wCFG`
- `decimals = 18`

File: test/fork/Fork.t.sol [hunk 3]
```diff
@@ -29,6 +29,16 @@ interface IAxelarContract {
     function contractId() external view returns (bytes32);
 }
 
+interface IAuth {
```
Comment (hieronx) [inline test/fork/Fork.t.sol:32]: This can be imported from `src/interfaces/IAuth.sol` right?

**Review by hieronx — DISMISSED** [review]

File: test/fork/Fork.t.sol [hunk 4]
```diff
@@ -54,6 +60,8 @@ struct Deployment {
 contract ForkTest is Test {
     using stdJson for string;
 
+    address constant UNSET_ADDRESS = address(32);
```
Comment (peculiarity) [inline test/fork/Fork.t.sol:63]: Probably you can add a comment about this special `address(32)` :)

Comment (AStox) [reply test/fork/Fork.t.sol:63]: Thats a good idea!

Comment (peculiarity) [reply test/fork/Fork.t.sol:64]: But why is it exactly 32?

File: test/fork/Fork.t.sol [hunk 5]
```diff
@@ -433,31 +439,74 @@ contract ForkTest is Test {
...
             abi.decode(deployments[id].parseRaw(".isTestnet"), (bool))
         );
     }
+
+    function testIntegrations() public {
+        if (vm.envOr("FORK_TESTS", false)) {
+            for (uint256 i = 0; i < deployments.length; i++) {
+                uint256 chainId = _getUint256(i, ".chainId");
+                address cfg;
+                try this._getAddress(i, ".integrations.cfg") returns (address _cfg) {
```
Comment (hieronx) [inline test/fork/Fork.t.sol:479]: This try catch could just be added within the `getAddress` method

**Review by peculiarity — APPROVED** [review]

**Review by ilinzweilin — APPROVED** [review]

**Pull request merged**